docs/cep/getting-started-stream-workers.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 10
+title: Getting Started with Stream Workers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to create a stream worker that reads data added to a collection and sends the data to a stream. After you create the stream worker, there are instructions for testing it. 
+
+## Create a Stream Worker
+
+Refer to the [Stream Workers](/cep/index.md) section for an explanation of how stream workers function.
+
+1. [Log in to your Macrometa account](https://auth-play.macrometa.io/).
+1. Click **Stream Workers**.
+1. In the Editor tab, define the stream worker. Copy and paste the following code block in the code editor on the Editor tab.
+
+```sql
+@App:name('sample-cargo-app')
+@App:description('Basic stream worker to demonstrate reading data from a collection and sending it to a stream. The stream and collection are created automatically if they do not already exist.')
+@App:qlVersion('2')
+
+/**
+Testing the Stream Worker:
+    1. Open Stream `SampleCargoAppDestStream` in Console. The output can be monitored here.
+
+    2. Upload following data into `SampleCargoAppInputTable` C8DB Collection.
+        {"weight": 1}
```

Comment:
This format drives me bonkers. The users literally cannot copy and paste it and have the block work! Something to keep an eye on as we continue to improve the content.

Reply:
> This format drives me bonkers. The users literally cannot copy and paste it and have the block work! Something to keep an eye on as we continue to improve the content.

Agreed, when I open a JIRA to fix the sample app description, I'll have them fix the instructions as well.